CellFabric/Cell_Fabric_FinFET__Mock/fabric_CMC_NMOS.py

```diff
@@ -0,0 +1,218 @@
+import sys
+import json
+import argparse
+from os import system
+import gen_gds_json
+import gen_lef
+import itertools
+
+from cell_fabric import Via, Region, Canvas, Wire, Pdk
+from cell_fabric import CenterLineGrid, UncoloredCenterLineGrid, ColoredCenterLineGrid
+from cell_fabric import Grid, EnclosureGrid, SingleGrid, CenteredGrid
+
+
+class CanvasNMOS(Canvas):
```

Comment:
Is there a reason you are creating CanvasNMOS in every file? Are they not identical? Why not put CanvasNMOS in a separate file and import it in all the files you need to subclass it?

Reply:
They are identical for most of the cells except caps and resistors. I will modify that, however, Wenbin can use these cells for now. I will update these changes

Reply:
Already discussing the NMOS vs. Cap / Res fabric differences in another thread. Please remove the redundant CanvasNMOS at least. Marking as resolved.